library/css/components/footer.css

```diff
@@ -48,7 +48,6 @@
   flex-direction: column;
   justify-content: space-between;
   text-align: start;
-  width: 400px;
```

Comment:
This does not work in 1440px screen, we have extra space left to the right
![image](https://user-images.githubusercontent.com/76113183/216092852-e1594608-4029-44d9-bbcb-39ecea5207a2.png)